spec.bs

```diff
@@ -320,26 +421,243 @@ maps to an internal [=fenced frame config=] [=struct=].
+export for="exfiltration budget metadata">amount to debit</dfn>, which is a non-negative valid
+floating point number <span class=XXX>TODO: ref to float</span>.
+
+An <dfn export for=fencedframetype>exfiltration budget metadata reference</dfn> is a struct
+containing an <dfn export for="exfiltration budget metadata reference">origin</dfn>, which is an
+[=origin=]; and an <dfn export for="exfiltration budget metadata reference">amount to debit
+reference</dfn>, which is a mutable reference to a non-negative valid floating point number.
+<span class=XXX>TODO: are mutable references a thing in spec?</span>
+
+An <dfn export for=fencedframetype>embedder shared storage context</dfn> is a string.
+
+A <dfn export>partition nonce</dfn> is <span class=XXX>TODO: Specify the type for this.</span>
 
 A <dfn export>fenced frame config</dfn> is a struct with the following [=struct/items=]:
 
-* An <dfn for="fenced frame config">urn uuid</dfn>, a [=urn uuid=] string
-* <span class=XXX>TODO: Specify the other members</span>
+<dl export dfn-for="fenced frame config">
+  : <dfn>mapped url</dfn>
+  :: null, or a struct containing a [=fencedframetype/mapped url=] and a
+     [=visibility/visibility to embedder=]
+
+  : <dfn>container size</dfn>
+  :: null, or a struct containing a [=fencedframetype/size=] and a
+     [=visibility/visibility to content=]
+
+  : <dfn>content size</dfn>
+  :: null, or a struct containing a [=fencedframetype/size=] and a
+     [=visibility/visibility to embedder=]
+
+  : <dfn>interest group descriptor</dfn>
+  :: null, or a struct containing an [=fencedframetype/interest group descriptor=], a
+     [=visibility/visibility to embedder=], and a [=visibility/visibility to content=]
+
+  : <dfn>effective sandbox flags</dfn>
+  :: null, or a struct containing an [=fencedframetype/exhaustive set of sandbox flags=] and a
+     [=visibility/visibility to embedder=]
+
+  : <dfn>effective permissions</dfn>
+  :: null, or a struct containing an [=fencedframetype/exhaustive set of permissions=], a
+     [=visibility/visibility to embedder=], and a [=visibility/visibility to content=]
+
+  : <dfn>fenced frame reporter</dfn>
+  :: null, or a struct containing a [=fencedframetype/fenced frame reporter=], a
+     [=visibility/visibility to embedder=], and a [=visibility/visibility to content=]
+
+  : <dfn>exfiltration budget metadata</dfn>
+  :: null, or a struct containing an [=fencedframetype/exfiltration budget metadata=], a
+     [=visibility/visibility to embedder=], and a [=visibility/visibility to content=]
+
+  : <dfn>nested configs</dfn>
+  :: null, or a struct containing a sequence of [=fenced frame configs=] and a
+     [=visibility/visibility to embedder=]
+
+  : <dfn>embedder shared storage context</dfn>
+  :: null, or an [=fencedframetype/embedder shared storage context=]
+</dl>
+
+A <dfn export>fenced frame config instance</dfn> is a struct with the following [=struct/items=]:
```

Comment:
I think it would be good to have some sort of note or description about _who_ is responsible for converting a config into a config instance.

Reply:
Is this covered by the non-normative blurb:

```
Each time a <{fencedframe}> navigates to a [=fenced frame config=], it is instantiated as a new
[=fenced frame config instance=], which governs that particular context inside the [=fenced
navigable container/fenced navigable=].
```

or do you mean some additional text right next to this declaration? I imagine the text would be basically identical to that quoted bit.

Reply:
No I think this covers it, I must've forgotten about that para during my review.